content/sales.rst

```diff
@@ -0,0 +1,152 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describes operational Sales details.
+
+
+Product Demo
+============
+
+Please see the `DemoSession` wiki page for the demo session structure.
+
+
+Demo text template to use
+-------------------------
+
+The demo session content is similar to the documentation in that it is aimed at
+system administrators and other individuals involved.
+
+Please send your criteria of requirements so that we can
+better answer your questions in advance of the demo.
```

Comment:
answer the question and prepare the demo session to touch all your requirements.

Or tailor the demo for your needs, and don't waste time demoing things which you will not use/need